src/NuGet.Clients/PackageManagement.PowerShellCmdlets/Cmdlets/InstallPackageCommand.cs

```diff
@@ -99,6 +100,12 @@ protected override void ProcessRecordCore()
             {
                 await InstallPackageByIdAsync(Project, Id, ResolutionContext, this, WhatIf.IsPresent);
             }
+            catch (FatalProtocolException)
+            {
+                // Wrap FatalProtocolException coming from the server with a user friendly message
+                var error = String.Format(CultureInfo.CurrentUICulture, Strings.Exception_PackageNotFound, Id, Source);
+                Log(MessageLevel.Error, ExceptionUtilities.DisplayMessage(new FatalProtocolException(error)));
```

Comment:
also consider printing original exception message as it can be helpful to troubleshoot the failure.